silkworm/silkrpc/commands/rpc_api_test.cpp

```diff
@@ -245,7 +245,7 @@ class RequestHandler_ForTest : public silkworm::rpc::http::RequestHandler {
                            commands::RpcApi& rpc_api,
                            const commands::RpcApiTable& rpc_api_table,
                            std::optional<std::string> jwt_secret)
-        : silkworm::rpc::http::RequestHandler(socket, rpc_api, rpc_api_table, std::move(jwt_secret)) {
+        : silkworm::rpc::http::RequestHandler(socket, rpc_api, rpc_api_table, std::nullopt, std::move(jwt_secret)) {
```

Comment:
I would change this `std::nullopt` with an empty vector of allowed origins (see comments below on the same topic).

Reply:
If I use an empty vector here it means the request_handler class must get it and save it by value